src/patternfly/demos/Page/examples/Page.md

```diff
@@ -135,4 +137,14 @@ wrapperTag: div
     {{/card}}
   {{/page-main-section}}
 {{/inline}}
-```
\ No newline at end of file
+```
+
+### Context selector in sidebar
+```hbs isFullscreen
+{{> page-template page-template--id="nav-basic-example" page-template-sidebar--hasContextSelector='true'}}
+```
+
+### Context selector expanded in sidebar
+```hbs isFullscreen
+{{> page-template page-template--id="nav-basic-example" page-template-sidebar--hasContextSelector='true' page-template-sidebar--hasContextSelectorExpanded='true'}}
```

Comment:
I would probably use `page-template-context-selector--IsExpanded` instead of `page-template-sidebar--hasContextSelectorExpanded`. Or update `page-template-sidebar--hasContextSelectorExpanded` to `page-template-sidebar--IsContextSelectorExpanded`, but I think it makes more sense as `page-template-context-selector--IsExpanded`

```suggestion
{{> page-template page-template--id="nav-basic-example" page-template-sidebar--hasContextSelector='true' page-template-context-selector--IsExpanded=true}}
```